src/csharp/Microsoft.Spark/Interop/Ipc/JvmBridge.cs

```diff
@@ -80,7 +80,6 @@ private ISocketWrapper GetConnection()
             // new connection is delegated to these threads in a round robin fashion.
             // A deadlock can occur on the JVM if a new connection is scheduled on a
             // blocked thread.
```

Comment:
This comment block should follow semaphore.Wait

Reply:
Makes sense. Thanks.

Reply:
Pushed a fix. Please review.